app/config/locales/en.yml

```
@@ -82,7 +82,10 @@ en:
           middle_name: Client's middle name
           todays_date: Today's date
       new:
-        description_html: "<p>Provide some details about the client so we can send them a link to verify their pay information. We'll request the past %{pay_income_days} days of income, based on the client's application date.</p><p>If multiple members of the household earn income, send an invite link to each household member.</p>"
+        description_html:
+          ma: "<p>Provide some details about the client so we can send them a link to verify their pay information. We'll request the past %{pay_income_days} days of income, based on today's date.</p><p>If multiple members of the household earn income, send an invite link to each household member.</p>"
+          nyc: "<p>Provide some details about the client so we can send them a link to verify their pay information. We'll request the past %{pay_income_days} days of income, based on the client's application/recertification date.</p><p>If multiple members of the household earn income, send an invite link to each household member.</p>"
+          sandbox: "<p>Provide some details about the client so we can send them a link to verify their pay information. We'll request the past %{pay_income_days} days of income, based on the client's application/recertification date.</p><p>If multiple members of the household earn income, send an invite link to each household member.</p>"
```

Side note, but I think for these site-specific strings where we have only one customization, we should use the "default" key along with whichever site required the customization. E.g. in this case

```suggestion
        description_html:
          default: "<p>Provide some details about the client so we can send them a link to verify their pay information. We'll request the past %{pay_income_days} days of income, based on the client's application/recertification date.</p><p>If multiple members of the household earn income, send an invite link to each household member.</p>"
          ma: "<p>Provide some details about the client so we can send them a link to verify their pay information. We'll request the past %{pay_income_days} days of income, based on today's date.</p><p>If multiple members of the household earn income, send an invite link to each household member.</p>"

```